### PR TITLE
Visual studio cheat sheet

### LINKED ISSUE #3230: Visual Studio Cheat Sheet: Add more shortcuts
I think this cheat needs more. 

There are literally hundreds of shortcut key combinations for VS 2015 and it would be nice to see some of the more interesting/useful ones featured so this CheatSheet is of interest to more seasoned VS developers as well as beginners.

I'm happy to help.
## Thanks

IA Page:  http://duck.co/ia/view/visual_studio_cheat_sheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @gautamkrishnar

### PR DESCRIPTION
**Pull request by biscuitNinja**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [ ] New Instant Answer
  - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
  - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
  - [x] Enhancement: **`#3230: Fleshing out of Visual Studio Cheat Sheet`**
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**
###### Description of changes

A more comprehensive set of shortcut key combinations for use with Visual Studio
###### Related Issues or Discussions

Fixes #3230
###### People to notify

---

Instant Answer Page: https://duck.co/ia/view/visual_studio_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @gautamkrishnar

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/visual-studio.json [hunk 1]
```diff
@@ -1,97 +1,867 @@
...
+            "key": "[Ctrl] [Shift] [.]",
+            "val": "Zoom in"
+          }, {
+            "key": "[Ctrl] [Shift] [,]",
+            "val": "Zoom out"
+          }
+        ],
+        "Window": [
+          {
+            "key": "[Esc]",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/visual-studio.json]: remove the `[]`

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/visual-studio.json]: remove the `[]`

File: share/goodie/cheat_sheets/json/visual-studio.json [hunk 2]
```diff
@@ -1,97 +1,867 @@
...
+            "key": "[Ctrl] [F6]",
+            "val": "Next document window"
+          }, {
+            "key": "[Ctrl] [Tab]",
+            "val": "Next document window nav"
+          }, {
+            "key": "[Alt] [F6]",
+            "val": "Next pane"
+          }, {
+            "key": "[F6]",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/visual-studio.json]: Here too

**Comment by gautamkrishnar** [comment]
@biscuitNinja any update on this?

**Comment by biscuitNinja** [comment]
@gautamkrishnar not yet. As soon as I've half an hour to spare I'll tackle it. I've some time off later this week.

**Comment by gautamkrishnar** [comment]
@biscuitNinja thanks a lot for your contribution 😄 🎉

**Comment by biscuitNinja** [comment]
@gautamkrishnar .. thanks for merging in.
I picked this up again yesterday and culled three/four sections. Probably be another pull request tomorrow for a slightly shorter version?

**Pull request merged**